src/design-tokens/brand.yml

```diff
@@ -0,0 +1,139 @@
+imports:
+  - ./colors.yml
+props:
+  primary_brand_lighter:
+    value: "{!blue-lighter}"
+    type: color
+    category: brand
+  primary_brand_light:
+    value: "{!blue-light}"
+    type: color
+    category: brand
+  primary_brand:
+    value: "{!blue}"
+    type: color
+    category: brand
+  primary_brand_dark:
+    value: "{!blue-dark}"
+    type: color
+    category: brand
+  primary_brand_darker:
+    value: "{!blue-darker}"
+    type: color
+    category: brand
+  green_1:
+    value: "{!green-lighter}"
+    type: color
+    category: brand
+  green_2:
+    value: "{!green-light}"
+    type: color
+    category: brand
+  green_3:
+    value: "{!green}"
+    type: color
+    category: brand
+  green_4:
+    value: "{!green-dark}"
+    type: color
+    category: brand
+  green_5:
+    value: "{!green-darker}"
+    type: color
+    category: brand
+  purple_1:
+    value: "{!purple-lighter}"
+    type: color
+    category: brand
+  purple_2:
+    value: "{!purple-light}"
+    type: color
+    category: brand
+  purple_3:
+    value: "{!purple}"
+    type: color
+    category: brand
+  purple_4:
+    value: "{!purple-dark}"
+    type: color
+    category: brand
+  purple_5:
+    value: "{!purple-darker}"
+    type: color
+    category: brand
+  fuchsia_1:
+    value: "{!fuchsia-lighter}"
+    type: color
+    category: brand
+  fuchsia_2:
+    value: "{!fuchsia-light}"
+    type: color
+    category: brand
+  fuchsia_3:
+    value: "{!fuchsia}"
+    type: color
+    category: brand
+  fuchsia_4:
+    value: "{!fuchsia-dark}"
+    type: color
+    category: brand
+  fuchsia_5:
+    value: "{!fuchsia-darker}"
+    type: color
+    category: brand
+  orange_1:
+    value: "{!orange-lighter}"
+    type: color
+    category: brand
+  orange_2:
+    value: "{!orange-light}"
+    type: color
+    category: brand
+  orange_3:
+    value: "{!orange}"
+    type: color
+    category: brand
+  orange_4:
+    value: "{!orange-dark}"
+    type: color
+    category: brand
+  orange_5:
+    value: "{!orange-darker}"
+    type: color
+    category: brand
+  gray_1:
+    value: "{!gray-lighter}"
+    type: color
+    category: brand
+  gray_2:
+    value: "{!gray-light}"
+    type: color
+    category: brand
+  gray_3:
+    value: "{!gray}"
+    type: color
+    category: brand
+  gray_4:
+    value: "{!gray-dark}"
+    type: color
+    category: brand
+  gray_5:
+    value: "{!gray-darker}"
+    type: color
+    category: brand
+  text_dark:
+    value: "{!black-transparent-85}"
+    type: color
+    category: text-color
+  text_light:
+    value: "{!white-transparent-85}"
+    type: color
+    category: text-color
+  text_light_emphasis:
+    value: "{!white}"
+    type: color
+    category: text-color
+  text_action:
+    value: "{!blue}"
+    type: color
+    category: text-color
```

Comment:
One other thought: Should all these tokens be in the same file? Is there value in having them separate. (I did that just to test that it worked!)